src/Microsoft.AspNetCore.Mvc.Razor/Compilation/CompiledViewDescriptorProvider.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Razor.Hosting;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Compilation
+{
+    public abstract class CompiledViewDescriptorProvider
```

Comment:
The intent is to make it easier to write a custom `ViewsFeatureProvider`

Reply:
Sorry, **whom**